lib/run.go

```diff
@@ -171,13 +171,14 @@ func findCmdInPath(pathlist []string, cmd, prefix string) (string, error) {
 	}
 
 	for _, p := range pathlist {
-		ex, err := util.Exists(path.Join(prefix, p, cmd))
-		if err != nil {
+		_, err := os.Lstat(path.Join(prefix, p, cmd))
```

Comment:
Can we use [path.IsAbs()](https://golang.org/pkg/path/#IsAbs) for `if len(cmd) != 0 && cmd[0] == '/'` ?
@dgonyeo